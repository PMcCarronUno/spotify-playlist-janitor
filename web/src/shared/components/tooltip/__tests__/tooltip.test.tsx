import {
  act,
  fireEvent,
  getByTestId,
  getByText,
  render,
  waitFor,
} from "@testing-library/react";

import { Tooltip } from "..";

describe("<Tooltip />", () => {
  let container: HTMLElement;
  let props: any;

  beforeEach(() => {
    props = {
      content: "Tooltip Content",
      dataTooltipId: "test-tooltip",
      children: <div data-testid="tooltip-child-test-id">Tooltip Child</div>,
    };
    ({ container } = render(<Tooltip {...props} />));
  });

  it("should render Tooltip component", () => {
    expect(container.firstChild).toMatchSnapshot();
  });
});
