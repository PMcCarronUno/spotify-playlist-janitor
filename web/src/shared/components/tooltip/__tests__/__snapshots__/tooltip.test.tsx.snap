// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tooltip /> should render Tooltip component 1`] = `
<div
  data-testid="tooltip-child-test-id"
  data-tooltip-id="test-tooltip"
>
  Tooltip Child
</div>
`;
